docs/MANUAL_SETUP.md

```diff
@@ -18,6 +18,20 @@ avoided. (postgresql + postgis binaries on mac use a newer version of gdal that
 NOTE: installing postgresql, postgis and gdal as binaries is the preffered method of installation. Installing
 from source is a world of trouble you don't want to get into. Stick to brew.
 
+##### Artifactory npm repo config
+
+We currently publish the cffdrs_ts package to our internal artifactory instance. You need to configure credentials so npm can pull the cffdrs_ts package from this repo. To add the credentials to your global `~/.npmrc` run:
+```bash
+npm config set @psu:registry
+https://artifacts.developer.gov.bc.ca/artifactory/api/npm/pe1e-psu-npm-local/
```

Comment:
Is this URL supposed to be an argument to the `npm config set @psu:registry` command above?

Reply:
Yep, not sure how the space got in there! thanks

Reply:
Fixed